lib/lib-dynamodb/src/index.ts

```diff
@@ -4,5 +4,3 @@ export * from "./DynamoDBDocumentClient";
 // smithy-typescript generated code
 export * from "./commands";
 export * from "./pagination";
-
-export { NumberValueImpl as NumberValue } from "@aws-sdk/util-dynamodb";
```

Comment:
Is this export being removed expected?

Reply:
fixed in https://github.com/aws/aws-sdk-js-v3/pull/5456